tools/IceRpc.Protobuf.Tools/ProtocTask.cs

```diff
@@ -132,3 +140,17 @@ protected override void LogEventsFromTextOutput(string singleLine, MessageImport
     /// <inheritdoc/>
     protected override void LogToolCommand(string message) => Log.LogMessage(MessageImportance.Normal, message);
 }
+
+/// <summary>Converts a byte array to a hexadecimal string.</summary>
+public static class HexStringConverter
+{
+    public static string ToHexString(byte[] bytes)
+    {
+        var sb = new StringBuilder(bytes.Length * 2);
+        foreach (byte b in bytes)
+        {
+            sb.Append(b.ToString("x2"));
+        }
+        return sb.ToString();
+    }
+}
```

Comment:
Would ideally use `Convert.ToHexString` but this is not available in `ToolTasks` due to being net standard 2.0